recipes/glib/all/test_v1_package/conanfile.py

```diff
@@ -11,14 +11,15 @@ def build_requirements(self):
             self.build_requires("pkgconf/1.9.3")
 
     def build(self):
-        if self.settings.os != "Windows":
-            with tools.environment_append({"PKG_CONFIG_PATH": "."}):
+        if not tools.cross_building(self) and self.settings.os != "Windows":
```

Comment:
it was working fine for cross-build to macOS M1, so I don't understand.

Reply:
Was an issue building in Yocto. I'm not if your recent changes would have effected that. And unfortunately, I can't even reproduce because `pkgconf` is failing. I'll need to fix that before I can test with this check is no longer necessary.

Reply:
Wait, nevermind. I had a very unfortunate typo in that cross-compile command. I can confirm that it fails to link the test package when I cross-compile due to these specific linker errors:

```
ld: cannot find -lselinux: No such file or directory
ld: cannot find -lsepol: No such file or directory
```

Reply:
I bet `libselinux` needs updated.

Reply:
No, this works in the normal test package. I think this must be related to the transitive dependencies not being passed on from the old CMake generator. I can still remove it though, since the recipe won't automatically try to build the v1 test package locally.